### PR TITLE
Improving previously added CombC Example

### PR DESCRIPTION
**Pull request by tmm88**

### CODE REVIEW

File: HelpSource/Classes/CombC.schelp [hunk 1]
```diff
@@ -64,20 +64,25 @@ code::
...
-(
-s.options.memSize= 8192*2;
+s.options.memSize= 8192*200; 
 s.reboot;
+
+(
+SynthDef(\longDelayLine, {
+	var in, delA, delB, delC, out;
+	in = SoundIn.ar(1);
+	delA=CombC.ar(in, 8, 8, 1, 0.3, 0);
```
Comment (snappizz) [inline HelpSource/Classes/CombC.schelp]: whitespace -- this should be `delA = CombC`

File: HelpSource/Classes/CombC.schelp [hunk 2]
```diff
@@ -64,20 +64,25 @@ code::
 { CombN.ar(Decay.ar(Dust.ar(1,0.5), 0.2, WhiteNoise.ar), 0.2, 0.2, 3) }.play;
 
 // large delay lines
-(
-s.options.memSize= 8192*2;
+s.options.memSize= 8192*200;
```
Comment (snappizz) [inline HelpSource/Classes/CombC.schelp]: whitespace around `=`

File: HelpSource/Classes/CombC.schelp [hunk 3]
```diff
@@ -64,20 +64,25 @@ code::
 { CombN.ar(Decay.ar(Dust.ar(1,0.5), 0.2, WhiteNoise.ar), 0.2, 0.2, 3) }.play;
 
 // large delay lines
-(
-s.options.memSize= 8192*2;
+s.options.memSize = 8192 * 200;
```
Comment (redFrik) [inline HelpSource/Classes/CombC.schelp]: no no no! do not commit this example.

Comment (tiagmoraismorgado) [inline HelpSource/Classes/CombC.schelp]: why specifically do you point that out. or what would you like to see changed?

**Comment by joshpar** [comment]
This really should be a power of 2, and I would also add a comment about what you are doing and why.
Thanks!

**Comment by tiagmoraismorgado** [comment]
thanks josh, i already fixed that out

File: HelpSource/Classes/CombC.schelp (hunk 3 above)
Comment (redFrik) [inline HelpSource/Classes/CombC.schelp]: sigh. we've been through this before.  have you already forgotten?  this is the third time you're suggesting changes to that example https://github.com/supercollider/supercollider/pull/2157#issuecomment-223759221 https://github.com/supercollider/supercollider/pull/2754#issuecomment-283316402
please be satisfied with what is there.  it is good. what you're suggesting here is not.  and if you can't explain why it's so bad then please go and do some supercollider studying and come back when you understand.

**Comment by snappizz** [comment]
current problems with this PR:

- there now *two* examples, one using a synthdef and one using `Function:play`. why? can we just have the synthdef one?
- messy commit history with a series of comments all with the same default "Update CombC.schelp" commit message
- badly written code -- aforementioned whitespace issues, and then `o=s.options; o.memSize; o.memSize = 2.pow(21);` which combines several statements on the same line, uses global variables inappropriately, and contains an extraneous statement

......

**Comment by telephon** [comment]
using function.play is good in examples, often clearer than SynthDef

**Comment by jamshark70** [comment]
> messy commit history with a series of comments all with the same default "Update CombC.schelp" commit message

This is probably the result of clicking "Edit" next to the file in the web interface, and then failing to change the default commit message.

That "Edit" button can be bad news...

File: HelpSource/Classes/CombC.schelp [hunk 4]
```diff
@@ -63,21 +63,33 @@ code::
...
 // large delay lines
-(
-s.options.memSize= 8192*2;
+o=s.options; o.memSize; o.memSize = 2.pow(21);
 s.reboot;
+
+(
+SynthDef(\longDelayLine, {
+	var in, delA, delB, delC, out;
+	in   = SoundIn.ar(1);
```
Comment (vivid-synth) [inline HelpSource/Classes/CombC.schelp]: Why 1 instead of 0 here?

File: HelpSource/Classes/CombC.schelp [hunk 5]
```diff
@@ -63,6 +63,17 @@ code::
 // used as an echo.
 { CombN.ar(Decay.ar(Dust.ar(1,0.5), 0.2, WhiteNoise.ar), 0.2, 0.2, 3) }.play;
 
+({
+ 	var in, del1, del2, del3, del4;
```
Comment (vivid-synth) [inline HelpSource/Classes/CombC.schelp]: Also, isn't this the example from below the "// large delay lines" comment? If so we shouldn't put this code before that note

**Comment by brianlheim** [comment]
@tmm88 do you plan to continue work on this PR?

**Comment by brianlheim** [comment]
Closing this stale PR

**Pull request closed without merging**